### DISCUSSION TITLE
⭐️ Starstruck Exchange ⭐️

### DISCUSSION THREAD

**Discussion by gelfik** [Schweinepriester/github-profile-achievements#694] [General]
Star mine and I will do the same.
https://github.com/gelfik/gelfik

**Comment by itspasindu** [comment]
Star my project and comment below, I will star yours too:

https://github.com/itspasindu/isipathana-college-demo-site

**Reply by gelfik** [reply]
done :)

**Reply by elijahpereira** [reply]
done @itspasindu https://github.com/elijahpereira/elijahpereira.github.io

**Comment by treeplate** [comment]
mine is treeplate/syllad

**Comment by PARKER-X** [comment]
@gelfik  Done!
https://github.com/PARKER-X/BarBar

**Reply by gelfik** [reply]
Done

**Comment by elijahpereira** [comment]
@gelfik https://github.com/elijahpereira/elijahpereira.github.io done!

**Reply by gelfik** [reply]
> @gelfik https://github.com/elijahpereira/elijahpereira.github.io done!

Done

**Comment by ArielMAJ** [comment]
@gelfik I starred you, please star my repo too: https://github.com/ArielMAJ/test.me
Feel free to contribute as well, there are many somewhat "simple" issues.

**Reply by gelfik** [reply]
Done

**Reply by ArielMAJ** [reply]
Ty!

**Comment by Veercodeprog** [comment]
You can take part in ongoing [coding problem](https://github.com/Veercodeprog/Data-Structures-implementation/discussions/categories/coding-discussion-share-your-solution-with-best-time-complexity-approach) discussion.

A daily problem will be posted, and the answer with the most efficient time complexity submitted first will be selected as the accepted solution.

**Reply by ArielMAJ** [reply]
Ok

**Comment by Veercodeprog** [comment]
https://github.com/Veercodeprog/full-stack-assignment. done

**Comment by xommmax** [comment]
HI everyone!
[Flutter Guidelines](https://github.com/4lfant/flutter-guidelines) repository contains my personal recommendations for policies, procedures, and standards regarding mobile development. I collected all my knowledge in this document.
Please star it 🥰
